docs/reference/security/authentication/configuring-file-realm.asciidoc

```diff
@@ -1,28 +1,33 @@
-You don't need to explicitly configure a `file` realm. The `file` and `native`
+The `file` and `native`
```

Comment:
IntelliJ insisted on removing some extra spaces making the diff bigger than it is. The only real changes are:

* This paragraph + "If you configure the realm chain manually, you must include the `file` realm for it to be used."
* This [line](https://github.com/elastic/elasticsearch/pull/110100/files#diff-2875c081b3aeff17ad892d195d4a962d2a30c808c9c3a9727238e8e502518ae0R27)